changelog/README.md

```diff
@@ -8,6 +8,10 @@ For more details, see [Environments](../guidelines/environments.md).
 
 # Changelog
 
+## 22th August 2022
```

Comment:
```suggestion
## 23th August 2022
```

Reply:
Note in English it should be 22nd (short for twenty-seco**nd**) or 23rd (short for twenty-thi**rd**). I suppose we could drop it all together, e.g. "23 August 2022", but it's not quite as easy to read and we would have to make the change in all Changelogs in all APIs for consistency.

Reply:
My mistake and yes, we should have same style everywhere.
Fixed.